### PR TITLE
PR: Add a standalone TranslationManager to be used outside of jlab plugins

### PR DESCRIPTION
**Pull request by goanpeca**
## Code changes

Adds a standalone TranslationManager so examples in the repo that do not rely on plugins can use translations.

## User-facing changes

None

## Backwards-incompatible changes

None

### CODE REVIEW

File: packages/translation/src/manager.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  private _connector: TranslatorConnector;
+  private _currentLocale: string;
+  private _domainData: any = {};
+  private _englishBundle = new Gettext();
+  private _languageData: any;
+  private _translationBundles: any = {};
+}
+
+export { TranslationManager };
```
Comment (blink1073) [inline packages/translation/src/manager.ts:71]: We typically put the export with the class (`export class Foo`)

**Comment by blink1073** [comment]
Mind adding an entry to the [PR labeler config](https://github.com/jupyterlab/jupyterlab/blob/7d39e5a7baad3d7362703a6861290dc3f52b9e84/.github/labeler.yml) for translation so it gets picked up?

File: packages/translation/src/manager.ts (hunk 1 above)
Comment (goanpeca) [reply packages/translation/src/manager.ts:71]: Fixed.

**Comment by goanpeca** [comment]
>Mind adding an entry to the PR labeler config for translation so it gets picked up?

Done!

**Comment by goanpeca** [comment]
Made the fixes @blink1073, I do not think the failures are related to the changes in this PR.

**Pull request merged**